src/components/randomQuotes.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+class DisplayRandomQuotes {
```

Comment:
This is not how you create a React class component. Please check the documentation or any previous work